experimental/examples/chat-mpp/shared/src/commonMain/kotlin/Messages.kt

```diff
@@ -43,14 +43,20 @@ internal inline fun Messages(messages: List<Message>) {
                 ChatMessage(isMyMessage = message.user == myUser, message)
             }
         }
-//        items(messages, key = { it.id }) { message -> //TODO not working in JS
-//            ChatMessage(isMyMessage = message.user == myUser, message)
-//        }
+        if (displayStub) {
+            item {
+                // TODO temporary workaround here
+                // Stub if temporary workaround for better Compose display with SwiftUI interop
+                // For now, Compose view is bigger, then it should be.
+                Box(Modifier.height(250.dp))
```

Comment:
To make it easier for now, I used a little workaround. It corrects the size of ScrollableView content. In the future, we will calculate the size of Compose View correctly

Reply:
Fixed. Now, we don't need this workaround anymore.
This Box now sizes to the height of SwiftUI TextField only.